packages/core/src/components/chip/_chip.scss

```diff
@@ -0,0 +1,70 @@
+@import '../../global/variables';
+
+$ray-chip-border-width: 1px;
+$ray-chip-height: $ray-spacing-xs * 2 + 1rem;
+
+.#{$ray-class-prefix}chip {
+  @include no-select;
+  border: $ray-chip-border-width solid $ray-color-blue-50;
+  background-color: $ray-color-white;
+  padding: $ray-spacing-xs $ray-spacing-md;
+  border-radius: $ray-chip-height;
+  color: $ray-color-blue-50;
+  display: inline-block;
+  cursor: pointer;
+
+  &:not(:last-child) {
+    margin-right: $ray-spacing-xs;
+
+    [dir='rtl'] & {
+      margin-right: 0;
+      margin-left: $ray-spacing-xs;
+    }
+  }
+
+  @media (hover: hover) {
+    &:hover {
+      background-color: $ray-color-blue-10;
+      color: $ray-color-blue-50;
+    }
+  }
+
+  @include ray-breakpoint(desktop) {
+    &:hover {
+      background-color: $ray-color-blue-10;
+      color: $ray-color-blue-50;
+    }
+  }
+
+  .#{$ray-accessibility-class} &:focus {
```

Comment:
need to add

```css
outline: 0;
```

to remove this effect:

![image](https://user-images.githubusercontent.com/6812331/61900986-bb55fb80-aeec-11e9-979f-1ea167813323.png)